l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -109,3 +113,35 @@ def open(self, mode: str = "r", cached: bool = True, cache_size_bytes: int = int
         self.frames = self.root[FRAME_ARRAY_KEY]
         self.agents = self.root[AGENT_ARRAY_KEY]
         self.scenes = self.root[SCENE_ARRAY_KEY]
+
+    def __repr__(self) -> str:
+        fields = [
+            "Num Scenes",
+            "Num Frames",
+            "Num Agents",
+            "Total Time (hr)",
+            "Avg Frames per Scene",
+            "Avg Agents per Frame",
+            "Avg Scene Time (sec)",
+            "Avg Frame frequency",
+        ]
+        if len(self.agents) > 1:
+            times = self.frames[1:10]["timestamp"] - self.frames[0:9]["timestamp"]
```

Comment:
that should fail when there are less than 10 frames right?

Reply:
the only cases when this fails is when there are 0 or 1 frames (mean fails). 10 is just a small random number to avoid loading a lot of data. I can put a constant there, but It has no semantical meaning so I'm not sure about it...a comment maybe?

Reply:
`if len(self.agents) > 1 and if len(self. frames) > 9:`?

Alternatively num_sample = min(len(self. frames), 9) ...

Reply:
btw, just noticed that it should be frames and not agents

Reply:
address in last commit, agents was a mistake :)